set.mm

```diff
@@ -215215,6 +215183,40 @@ is an important property of monoids (see ~ mndid ), and therefore also for
       USVCURNECVBUOBUMUTUMDCEUTERVAUNUMUTEUMDUCUDUEUFUHUIUJUK $.
   $}
 
+  ${
+    $d B x $.  $d L x $.  $d R x $.  $d .+ x $.
+    lidrideqd.l $e |- ( ph -> L e. B ) $.
+    lidrideqd.r $e |- ( ph -> R e. B ) $.
+    lidrideqd.li $e |- ( ph -> A. x e. B ( L .+ x ) = x ) $.
+    lidrideqd.ri $e |- ( ph -> A. x e. B ( x .+ R ) = x ) $.
+    $( If there is a left and right identity element for any binary operation
+       (group operation) ` .+ ` , both identity elements are equal.
+       Generalization of statement in [Lang] p. 3: it is sufficient that "e" is
+       a left identity element and "e``" is a right identity element instead of
+       both being (two-sided) identity elements.  (Contributed by AV,
```

Comment:
I think it is a bit verbose. No reference is necessary, and the comment could be shortened to something more punchy like

> In a magma, a left identity and a right identity are equal.

Your choice.

Reply:
> I think it is a bit verbose. No reference is necessary, 

Maybe not necessary, but I think it is always a good idea to add a refeence if there is one.

> and the comment could be shortened to something more punchy like
> 
> > In a magma, a left identity and a right identity are equal.

Since it is not required that there is a magma (the operation is closed), I would not like to change the comment.